internal/controller/account_resolver.go

```diff
@@ -62,8 +63,10 @@ func (bar *BOPAccountIdResolver) MapClientIdToAccountId(ctx context.Context, cli
 	if err != nil {
 		return "", "", "", err
 	}
+	uuid := uuid.NewString()
```

Comment:
Can you rename this to `requestID`?  I think that communicates what it is being used for a bit better.

We also need to log the requestID before we make the request.  The request id needs to be logged as a field named "request_id".

You could move the request id generation up above line 54 and then set the client_id and request_id field in one shot like this:

```
logger := logger.Log.WithFields(logrus.Fields{"client_id": clientID, "request_id": requestID})
```